b_lambda_layer_common/source/python/exceptions/exception_mapper.py

```diff
@@ -54,6 +55,8 @@ class ExceptionMapper:
 
     @staticmethod
     def map_and_raise(exception: Union[str, Dict[str, Any], Exception]) -> None:
+        logger.info(f'Trying to map error: {exception}')
```

Comment:
Add a dot or preferably 3 dots.

Reply:
No need for 3 dots IMO, doesn't change the message meaningfully